gravity/util/__init__.py

```diff
@@ -87,3 +89,18 @@ def process_property(key, value, depth=0):
             value_sep = " "
         description = f"{description}\n{extra_white_space}{comment}{key}:{value_sep}{default}\n"
     return description
+
+
+def http_check(bind, path):
+    if bind.startswith("unix:"):
+        if not requests_unixsocket:
+            gravity.io.exception(
+                "The requests-unixsocket Python library is required to perform http checks on UNIX sockets, and it "
+                "does not appear to be installed")
```

Comment:
You could fall back to `subprocess.check_call(['curl', '--socket', ...])`, that's likely to be installed, and if that fails emit an error?